ashley/src/main/kotlin/ktx/ashley/listeners.kt

```diff
@@ -0,0 +1,26 @@
+package ktx.ashley
+
+import com.badlogic.ashley.core.Entity
+import com.badlogic.ashley.core.EntityListener
+
+/**
+ * An [EntityListener] only listening to the [entityAdded] event.
+ *
+ * A [Unit] implementation is provided for the [entityRemoved] event, as it is ignored.
```

Comment:
`[Unit] implementation` might not be clear for the users. I'd go with a `no-op` or `empty implementation`.

Reply:
Fixed in https://github.com/libktx/ktx/pull/410/commits/a2cdbcdf9ebcd52bdb467d5ad43b43e94096e964!
I chose to go with `empty implementation` as it seems less ambiguous to me.